### PR TITLE
initial slots for mass spec class

### PR DESCRIPTION
**Pull request by aclum**

### CODE REVIEW

File: src/schema/basic_slots.yaml [hunk 1]
```diff
@@ -33,10 +33,32 @@ slots:
...
+    description: Captures the primary mass analyzer used on the mass spectrometry run. 
+  ionization_source:
+    domain: MassSpectrometry
+    range: IonizationSourceEnum
+    description: Captures the ionization source used on the mass spectrometry run. 
+  mass_spectrum_collection_mode:
+    domain: MassSpectrometry
+    range: MassSpectrumCollectionModeEnum
+    description: Captures the type of mass spectrum data aquired
+    comments: Profile vs Centroid
```
Comment (aclum) [inline src/schema/basic_slots.yaml:57]: comments are multivalued, format as a list.

Comment (aclum) [reply src/schema/basic_slots.yaml:57]: consider removing comments once the enum exists.

File: src/schema/nmdc.yaml [hunk 2]
```diff
@@ -164,7 +164,13 @@ classes:
     exact_mappings: CHMO:0000470
     slots:
       - acquisition_category
+      - acquisition_strategy
```
Comment (aclum) [inline src/schema/nmdc.yaml:167]: @aclum fix exact_mapping to a list

File: src/schema/basic_slots.yaml [hunk 3]
```diff
@@ -33,10 +33,32 @@ slots:
     range: AcquisitionCategoryEnum
     description: Captures which type of acquisition category was on a mass spectrometry run.
     comments: MS vs MSn
+  acquisition_strategy: 
+    domain: MassSpectrometry
+    range: AcquisitionStrategyEnum
+    description: Captures which type of acquisition strategy was on a tandem mass spectrometry run.
+    comments: DIA vs DDA
```
Comment (corilo) [inline src/schema/basic_slots.yaml]: remove comments

File: src/schema/nmdc.yaml [hunk 4]
```diff
@@ -1768,10 +1775,62 @@ enums:
...
   ResolutionCategoryEnum:
     permissible_values:
       high: { }
       low: { }
+  
+  MassAnalyzerEnum:
+    permissible_values:
+      TOF:
+        aliases:
+          - Time Of Flight
```
Comment (corilo) [inline src/schema/nmdc.yaml]: flip aliases and permissible value

File: src/schema/nmdc.yaml [hunk 5]
```diff
@@ -1768,10 +1796,63 @@ enums:
...
     permissible_values:
       high: { }
       low: { }
+  
+  MassAnalyzerEnum:
+    permissible_values:
+      time_of_flight:
+        aliases:
+          - TOF
+      quadruppole:
```
Comment (corilo) [inline src/schema/nmdc.yaml:1818]: fix speling

File: src/schema/nmdc.yaml [hunk 6]
```diff
@@ -1768,10 +1796,63 @@ enums:
...
+        aliases:
+          - Quad
+      Orbitrap:
```
Comment (corilo) [inline src/schema/nmdc.yaml:1821]: add aliases Orbi

**Review by anastasiyaprymolenna — APPROVED** [review]

**Comment by aclum** [comment]
@corilo  I don't think i migration is needed for this PR since this is all additive (new slots) but it is dependent on the remodeling of OmicsProcessing. @eecavanna how do you want these labled?

**Comment by eecavanna** [comment]
Hi @aclum, based on my understanding of the situation, this is how I'd like it labeled/documented:

1. Change which label it has:

   ```diff
   - migration-needed
   + no-migration-needed
   ```

2. Add a description that includes:

   ```md
   #### No migration needed

   No migration is needed because these schema changes only _add_ slots, and all
   of those slots are _optional_ (i.e. documents lacking those slots are still valid).

   #### Dependencies

   Do not merge until `OmicsProcessing` has been remodeled (i.e. until the following
   PR has been merged):

   (@aclum - add the link here)
   ```

**Pull request merged**